webpack/move_to_pf/OptionTooltip/__tests__/OptionTooltip.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import OptionTooltip from '../';
+
+describe('OptionTooltip', () => {
```

Comment:
You could wrote much simpler tests if you would use the test-helper `testComponentSnapshotsWithFixtures`:
https://github.com/Katello/katello/blob/master/webpack/move_to_pf/test-utils/testHelpers.js#L31

Example using it:
https://github.com/Katello/katello/blob/fd8c2c012d62055b052a08820b37c76f35fef44e/webpack/move_to_foreman/components/common/table/components/TableBody.test.js

Reply:
Went ahead and refactored the test.  Thank you for pointing me to the helpers.